web/angular-wallet/src/app/login/login.component.scss

```diff
@@ -1,63 +1,93 @@
-$media-querie-mobile: 0;
-$media-querie-tablet: 768px;
-$media-querie-desktop: 992px;
-$media-querie-screen:  1200px;
+@import "src/@fuse/scss/fuse";
+
+$media-query-mobile: 0;
+$media-query-tablet: 768px;
+$media-query-desktop: 992px;
+$media-query-screen: 1200px;
```

Comment:
Angular material has a layout module for doing responsive stuff. https://material.angular.io/cdk/layout/overview

Might be worth looking into, as I can see this not scaling well.

Reply:
You're right.  These media queries are 'legacy' code